### PR TITLE
testnet3: Adjust to new specs and dependencies

### PR DESCRIPTION
**Pull request by jleni**
- [x] Updated `signer` tests
- [x] Updated wasm tests ~~(blocked because of dependencies issue)~~

### CODE REVIEW

**Comment by rllola** [comment]
The tests has been updated for the `signer` crate but not the wasm test. Like discussed, the dependencies added in `forest` doesn't allow WASM compilation.

**Comment by jleni** [comment]
We are temporarily pointing to this PR https://github.com/ChainSafe/forest/pull/417
We should update tags once it gets merged.

File: examples/manual_testvectors.json [hunk 1]
```diff
@@ -72,22 +72,7 @@
...
-            "from": "f00",
-            "nonce": 1,
-            "value": "100000",
-            "gasprice": "2500",
-            "gaslimit": 25000,
-            "method": 0
-        },
-        "encoded_tx_hex": "884b00ffffffffffffffffff014200000144000186a0430009c41961a80040"
+        "encoded_tx_hex": "8900430080084300c10d0144000186a0430009c41961a80040"
     },
```
Comment (rllola) [inline examples/manual_testvectors.json:76]: See issue https://github.com/Zondax/filecoin-rs/issues/143

File: examples/wasm_node/test/test.spec.js [hunk 2]
```diff
@@ -197,3 +197,31 @@ describe("Test for utils.js", function() {
...
+  const message_digest = utils.getDigest(Buffer.from(message, 'hex'));
+  const prvkey = "f15716d3b003b304b8055d9cc62e6b9c869d56cc930c3858d4d7c31f5f53f14a";
+
+  // Get hex signature in the format (R,S)
+  let signature = secp256k1.ecdsaSign(message_digest, Buffer.from(prvkey, 'hex'));
+
+  console.log(Buffer.from(signature.signature).toString('hex'))
+  console.log(signature)
+});
+
```
Comment (rllola) [inline examples/wasm_node/test/test.spec.js:224]: why twice ?

**Pull request merged**